core/src/main/resources/springTOMCATBTM.xml

```diff
@@ -16,8 +16,8 @@
 		class="nl.nn.adapterframework.jta.btm.BtmJtaTransactionManager"
 		depends-on="btmConfig"
 		>
-		<property name="statusFile" value="${transactionmanager.log.dir}/status.txt" />
-		<property name="uidFile"  value="${transactionmanager.log.dir}/tm-uid.txt" />
+		<property name="statusFile" value="${transactionmanager.log.dir}/BTM/status.txt" />
+		<property name="uidFile"  value="${transactionmanager.log.dir}/BTM/tm-uid.txt" />
```

Comment:
Dit is niet handig, sommige mensen maken hier nu al gebruik van. Maken de tests hier gebruik van? Dan zou ik daar verwachten dat die hier iets anders van maken...

Reply:
Omdat in de tests de hele tijd transaction managers wordt gewisseld, wou ik er zeker van zijn dat ze niet deze bestanden van elkaar gebruiken.

Ik zal kijken of dat nog lukt zonder deze spring contexts aan te passen maar iets wat alleen in tests wordt gebruikt.